cluster/util.py

```diff
@@ -130,3 +132,38 @@ def centroid(data, method=median):
     for i in range(len(data[0])):
         out.append(method([x[i] for x in data]))
     return tuple(out)
+
+def HDcentroid(data):
+    dict_words={}
+    dict_weight={}
+    words_per_user=10 #10 words per user. This value is not used.
```

Comment:
This line can be removed. The value is only used inside the scope of the following `for` loop, and only introduces "noise" here.

Reply:
i agree